website/0.8.4/src/site/markdown/Quickstart.md

```diff
@@ -406,7 +406,55 @@ Now, let\'s look at one of the partitions:
 
 ```
 # helix-admin.sh --zkSvr <zk_address> --listResourceInfo <clusterName> <partition>
```

Comment:
Good catch! Could you please also update the template here? Should be listPartitionInfo as well.

Reply:
sure

Reply:
> Good catch! Could you please also update the template here? Should be listPartitionInfo as well.

Fix the command template and corresponding example.